Tests.Template/changelist.md

```diff
@@ -0,0 +1,16 @@
+# Test Template Change List
+Any changes made to the *Tests.Template* files must be added to the appropriate section in this document.
+The version number in the *Tests.Template* header must also be updated to match.
+When the templates are used to create or update tests in a DSC Resource the version number of the template file used must be included.
+
+## integration_config_template.ps1
+### Version 1.0
```

Comment:
Can we use [semver](http://semver.org/)? Not sure if this will be particularly useful for test templates, but we will be moving with versioning of all resources into semver, so it would be nice to have it here as well for consistency.